### ISSUE TITLE
Shared Notes won't sync

### ISSUE THREAD

**Issue by ElArkk** [baumgarr/Nixnote2#458]
Nixnote 2.0.2 doesn't show notes from Evernote that were shared with me, just my personal notebooks. I've seen this issue was raised already some years ago, can I assume that this feature won't be implemented?

Regards
Arkadij

**Comment by robert7** [comment]
TLDR: Shared notebooks should work..

As you maybe noticed, the original maintainer isn't available now.. It someway happened, that I took over.. There is updated version of the app ("[v.2.1](https://github.com/robert7/nixnote2/releases)") in [my fork](https://github.com/robert7/nixnote2). So far I did mainly stabilization/bugfixing and added few new features. 

I don't use shared notebooks myself. And I also didn't touched the code which is handling it (and also don't have any such plans for future :) ). But I tried it few times and it worked fine.
Yesterday I created new notebook in one of my test accounts, added few random notes, shared it with another test account of me and it worked fine.

I would recommend:
* check using web client, if the sharing is working correctly (in the web client)
* make sure sync is working correctly (e.g. by adding/changing something in main notebook and checking in web client).
* you may try the new version and eventually [post log](https://github.com/robert7/nixnote2/wiki/HowTo---Add-log-file-to-the-github-issue)

![screen_20180920_01](https://user-images.githubusercontent.com/1135218/45838411-f3f7d800-bd11-11e8-85d1-8928c511a428.png)
![screen_20180920_02](https://user-images.githubusercontent.com/1135218/45838425-f9edb900-bd11-11e8-87ed-82c2ce92529b.png)
